script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_5_3.txt

```diff
@@ -8,7 +8,7 @@ There used to be a playroom on the second floor of the west wing...
 "Oh. So you remember it."
 <Page4>
 "But, my apologies.
-The large rooms such as the playroom, the library, and the salon have been locked since Akiha-sama became the head of the family.
+The large rooms such as the playroom, the library, and the entertainment room have been locked since Akiha-sama became the head of the family.
```

Comment:
> 秋葉さまがご当主になられてから、遊戯室や図書室、サロンといった大部屋は施錠されております

It's salon

Reply:
In English we say "[drawing room(UK) or parlor(US)](https://en.wikipedia.org/wiki/Drawing_room)". Salon usually means like an informal intellectual gathering or a beauty parlor as far as I'm concerned.